styles.css

```diff
@@ -0,0 +1,208 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+body {
+  background: #eceef1;
+}
+
+.page-header {
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 30px;
+  margin-top: 10px;
+}
+
+.signin-key {
+  height: 1em;
+  width: 0.8em;
+  color: #0077c5;
+  margin-right: 4px;
+  vertical-align: top;
+  display: inline-block;
+  background-image: url(https://plugin.intuitcdn.net/identity-authn-core-ui/1.2.9/images/icon-whitelock@3x.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+}
+
+.page-header button {
+  align-self: center;
+  background: #0077c5;
+  border-radius: 4px;
+  border: 1px solid #4c4c4c;
+  color: white;
+  padding: 7px 20px;
+  cursor: pointer;
+}
+
+.page-header ul {
+  margin-right: 10px;
+}
+
+.page-header ul li {
+  list-style-type: none;
+  font-size: 12px;
+  background: #eceef1;
+  line-height: 20px;
+  font-weight: 400;
+  color: #505050;
+}
+
+.page-main a {
+  font-size: 12px;
+}
+
+.page-main a,
+.page-header ul li a {
+  text-decoration: none;
+  color: #0077c5;
+}
+
+.intuit-logo {
+  width: 120px;
+  height: 120px;
+}
+
+.logos {
+  margin-top: -30px;
+}
+
+.page-main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.form-div {
+  width: 35%;
+  background: white;
+  height: inherit;
+  text-align: center;
+  margin-top: 10px;
+  padding-top: 30px;
+  border: 1px solid #d4d2d2;
+}
+
+.form-parag-one {
+  margin-top: 15px;
+  color: #4c4c4c;
+  font-size: 14px;
+}
+
+.form-div h2 {
+  color: #4c4c4c;
+  line-height: 1.1;
+  font-size: 24px;
+  font-weight: 300;
+}
+
+.form-label,
+form input {
+  margin-top: 30px;
+  width: 100%;
+  text-align: left;
+  margin-left: 40px;
+}
+
+.form-label {
+  color: #505050;
+}
+
+form input {
+  margin-top: 15px;
+  width: 85%;
+  margin-left: 5px;
+  height: 35px;
+  border-radius: 4px;
+  border: 1px solid #c1c5c8;
+}
+
+.form-submit button {
+  width: 85%;
+  height: 44px;
+  margin-left: 3px;
+  margin-top: 25px;
+  background: #0077c5;
+  border: none;
+  margin-bottom: 35px;
```

Comment:
Shorthand properties are CSS properties that let you set the values of multiple other CSS properties simultaneously. Using a shorthand property, you can write more concise (and often more readable) style sheets, saving time and energy.
For margin the sequence is ->
margin : top right bottom left.